app/views/health/sdh_case_management_notes/_form_js.haml

```diff
@@ -54,7 +54,7 @@
       var none = $(behaviorSelector("no activities"));
       var activities = $(behaviorSelector("activity"));
       var template = $(behaviorSelector("activity to copy"))[0];
-      var html = $(template).html().replace(/COPY/gi, activities.length);
+      var html = $(template).html().replace(/-1/gi, activities.length);
```

Comment:
-1 seems like maybe, a dangerous replacement pattern.  Maybe we should go with something like -999?

Reply:
I agree. Updated.